docs/spark-standalone.md

```diff
@@ -235,10 +235,11 @@ SPARK_WORKER_OPTS supports the following system properties:
 <tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
 <tr>
   <td><code>spark.worker.cleanup.enabled</code></td>
-  <td>false</td>
+  <td>true</td>
```

Comment:
Why have you changed this default value in the documentation? As I see it is still false.

https://github.com/apache/spark/blob/b15423361bc28c4cd2216683eb852fdbec3ea58f/core/src/main/scala/org/apache/spark/internal/config/Worker.scala#L31-L33

Reply:
sorry,i missed it.

Reply:
I believe attila meant to revert your change to the docs, not to change the default